### ISSUE TITLE
作文

### ISSUE THREAD

**Issue by ssfc** [ssfc/Note#40]
1997
Sample.
As an important cause for economic losses and health risks, tobacco consumption is always a hot issue in our society. According to Picture 3 and Picture 4, 200 billion US dollars are lost(被动语态) annually because of tobacco consumption, and meanwhile 3 million people worldwide lose their lives due to(固定搭配) smoking-related diseases.

To our great relief(固定搭配), there is a tendency to decline in tobacco production--the year between 1994 and 1995, as indicated by Chart 1, witnessed a drop from(固定搭配) 14.3 billion pounds to 14.3 billion pounds. The market shrank probably because more and more people were aware of(固定搭配) the harmful effects of tobacco and the quitted smoking.

If this trend could be maintained, the smoking-related problems would be significantly(副) reduced. However, it is difficult to achieve it. For one thing, the percentage of smokers is too large: according to a survey, there are 1.1 billion tobacco consumers in the world, accounting for approximately 20 percent of the world population. It would not be an easy task to persuade so many smokers, especially the heavy smokers, to give up their smoking habits. For another, tobacco industry is still a powerful engine for economic development. In this sense, only(倒装句) with concerted efforts of all people, can smoking-related problems be properly solved.

Mine
In the light of(固定搭配) Picture 3 and Picture 4, 200 billion US dollars are lost(被动语态) annually because of tobacco consumption, and meanwhile 3 million people worldwide lose their lives due to(固定搭配) smoking-related diseases. 

To our great relief(固定搭配), it can be predicted that the smoking-related problems would be significantly(副词) reduced in the near future, since tobacco production witnessed a drop from(固定搭配) 14.3 billion pounds in 1994 to 14.2 pounds in 1995. However, solving smoking-related problems still has a long way to go(固定搭配), since there are 1.1 billion smokers in the world, accounting for(状语) approximately 20 percent of world population as Picture 1 reveals. In this sense, only(倒装句) with concerted(形容词) efforts of all people, can smoking-related problems be properly(副词) solved.

1998

Sample
A hen is holding a notice promising that her eggs would be round and surely(副词) would contain egg shells, egg whites and yolks. Those promised qualities are only basic components of eggs. How ridiculous it is!

Under the funny appearance hides a serous social problem that there are many meaningless promises which only reflect the basic responsibilities of the promise makers. Manufacturing units guarantee to supply products of good quality; commercial enterprises swear to(动词) provide genuine commodities and enthusiastic services; administration departments pledge to perform their tasks effectively and fairly without taking any bribery.

Such promises are deceptive to the public. The hen and her likes should know that by dishonest words no one could survive the intense competition under market economy system. On one hand, people engaged in providing all kinds of services should honestly inform their customers of the services that can be provided because mutual trust is the foundation of business. They should remember the old saying, honesty is the best policy. On the other hand, necessary laws and regulations(并列) should be adopted and enforced to ensure a trust-worthy economic and social environment. Only in these ways(倒装), can both parties, service providers and those served, enjoy a healthy atmosphere of mutual trust. 

Mine
In this picture, a hen lying on the grass holds a notice in hand, asserting that its eggs would be round without angle, and surely(副词) contain egg shells, egg whites and yolks. This picture reveals that in contemporary society more and more companies tend to give all kinds of promise to their customers and welcome inspection from their customers. Although we admit it is a progress compared with the past, it should also be pointed out that some promises are basic responsibilities of commercial enterprises and there is no need to say out loudly.  

1999
Sample
There are two graph above. The first graph shows a continuous increase in American population from 1800 to 2000. The second graph demonstrates the extinction of 70 wild life species, among which 50 to 60 species disappeared from the planet between 1800 and 1900. It seems that the drop of the number wild species was closely related to the growth of population in the United States. In other words, the population explosion will have great destructive effect on the wild life.

The above argument makes sense logically for the following reason. The increasingly large population will be a great threat to all kinds of wild animals. People pollute the environment of animals natural habitat and even take up the home of many kinds of wild life. What is worse, the process of urbanization and industrialization will drive sorts of wild animals away from their shelters to other places where they cannot survive due to the harsh environment.  

To save endangered wild life from extinction, some actions need to be taken. First and foremost(连词), the human population should be controlled within a reasonable range through effective measures. Moreover, efforts should be made(被动) to create a pleasant living environment for the wild life. If effective actions are taken, the earth will be a habitable planet both for mankind and wildlife.

Mine
Population growth and species extinction are both hot topics in contemporary society. In the light of two provided picture, number of US people increased 250 million from 1800 to 2000, while 70 species no longer existed from 1600 to 1900, which apparently revealed that countries growing population accelerate the extinction of wild life. 

The above argument makes sense logically for the following reason. First of all, with more people means more industry that pollute the natural habitat of wild life. Besides, more and more forest become the houses of human beings. 

To save endangered wild life from extinction, some actions need to be taken. First, laws and regulation(并列) ought to be enforced and observed(被动并列) to keep the balance of nature by prohibiting blind exploitation. Moreover(连词), citizens should be educated to realize the value of wild life and voluntarily(副词) protect them. In summary, only(倒装句) with concerted efforts of all individuals, can our earth become a habitable planet both for mankind and wildlife.

2000
Sample
As we can see from the two pictures, great changes were taking place in the world commercial fishing. In 1900 there was only one ship in the sea rich in(固定搭配) fishes, while in 1995, there become only one fish in the sea crowd with(固定搭配) ships. The caption under the pictures demonstrates that it was the ever-developing fishing industry that caused the devastating effects on the fishing resources.

The pictures, at first, convey a deep concern about the ecological disaster which was brought about by the over-fishing driven by short-term commercial interests. They also imply a more worrisome consequences: the profit-oriented behavior may penetrate into other fields. People are likely to exploit other kinds of natural resources unlimitedly and blindly without considering the potential damage that can be done to nature and, more directly, to human beings.

To prevent such undesirable situation from happening, we should spare no efforts in(固定搭配) protecting natural resources. Laws and regulations should be enforced and observed(并列) to keep the balance of nature by prohibiting blind exploitation, such as over-fishing in this picture. In addition, people should be educated to realize the value of these resources and voluntarily protect them.

Mine
There are two pictures above. The first picture reveals there was only one fishing boat on the sea rich in fish, while the second picture demonstrate there become only one fish in the water crowed with ships 96 years later. The painter demonstrates that(宾语) the drop of the number of fish was closely related with the growth of the fishing boats. In other words, it was the ever-developing of commercial fishing that had great destructive effect on the fishing resources.

To prevent such undesirable situation from going on(套话), some actions need to be taken. First and foremost, the number of fishing boats ought to be controlled within a reasonable range through effective measures. Moreover, during the fishing close season, no one is allowed to catch fish, so that fish have enough time to mate and breed. To sum up(搭配), only with(倒装) concerted efforts of all individuals, can our earth become a habitable planet for both fish and mankind. 

2001
Sample
In this vivid(形容词) picture, an oil lamp with a happy smile twinkles in the dark. Around it is the caption reading “Love is similar to a light shining brighter in the darker place”. The picture means that love is considered more precious in more difficult situations or rather is more needed by people in urgent need of help.

Only when love is donated to the people in greater need, can it better achieve its value. I can exemplify this conclusion with the Poverty Alleviation Project. In its spirit of helping the poor in backward regions, many college students, upon graduation, choose to work in poor areas where they are most needed and their knowledge can be made better use of. Dedication of love is their choice. Seeing how excited the poverty-stricken people there are when they experience the change these dedicators bring about, we can know how much the devoted love and help are cherished.

The best way to show our love, in my opinion, is to follow the above mentioned example by giving love to the people suffering during the hours of darkness. Therefore, when we see someone in difficulty or in distress, we shouldn’t hesitate to offer our hands.

Mine
In the picture there is a oil lamp with a happy smile has the name of love. The darker the environment is, the lighter the oil lamp is. The painter demonstrate that love is precious in troubles. In my view, the best way to show love is helping others. 

2002
Sample
An American girl, dressed in Chinese traditional costume of a minority group, similes sweetly and radiantly(副词) in the photo above. The ethnic accessories such as jeweled headdress, beaded necklace, earrings with tassels and silk attire with ribbons have added oriental glamour to her beautiful appearance. Obviously, the girl who has great curiosity and interest in Chinese culture feels fairly(副词) satisfied with her new look.

In recent years, accompanied by(伴随) a global upsurge of Chinese craze, a new fashion trend of wearing Chinese tradition clothes has swept the world. More importantly, with rapid social development and frequent culture exchanges, a mounting number of foreigners begin to understand and accept splendid Chinese culture and some of them even flood into China to gain a deeper insight into it.

The phenomenon highlights the glamour of our culture, which has been universally accepted as the invaluable treasure of all human beings. The recent popularity of Chinese culture results from our great achievements during the last two decades, such as the re-unification of Hong Kong and Macao with China, the accession to WTO and so forth. How can one resist being appealed to such an energetic nation with an increasingly significant status as well as a long and colorful culture?

Mine
In this vivid(形容词) picture, an American girl with a sweet and radiant(形容词) smile is wearing traditional Chinese picture. Apparently(副词), this picture unveiled that(宾语) Chinese culture is accepted and loved by more and more people around the world.

This trend may due to the following reasons. In the first place, China has a history of 5000 years and Chinese culture is exceedingly(副词) valuable and desirable. In the second place, with the development of Chinese economy, our government and companies are able to let more people know Chinese culture. On the other hand, it should be confessed(被动) that Chinese tradition still have a long way to go before becoming the main culture of the world.

To sum up(固定搭配), due to the above mentioned reasons, we may safely arrive at the conclusion that our culture is fascinated by more and more people, and we ought to spare no effort to introduce it to more and more foreign friends.

2003
Sample
A flower thrives under the protection of the greenhouse, its branches tender and petals fresh. However, once removed from the shelter, it withers in the thunderstorm. The pictures indicate that an over-protected flower can never survive the harsh conditions(固定搭配).

Similarly, a child under complete protection of his parents can hardly survive hardship and harshness. This is particularly true of the children born in the 1980s, majority of them being the only child in their families. As the focus of parents and grandparents, the so-called little emperors or empresses used to be carefully protected. After they grow up and walk out of the protection, they painfully find they can not solve daily problems on their own. How worrisome it is that(感叹句) those fragile young people will have to be the pillar of society.

The tragedy of spoiled children is directly attributed to the failure of family education. The over-protective parents misunderstand the responsibility of parents as providing the child with a comfortable life. In fact, parents should treat their child as an individual who will become independent sooner of later and thus encourage the child to think and act independently. Until this right concept of parenting is widely accepted, we shall continue to have children like greenhouse flowers, beautiful but unable to stand up to life’s real challenges.

Mine
There are two pictures above. In the first picture, a flower is blooming in the green house, while in the second picture, the flower quickly died when exposed to the thunderstorm. These pictures unveiled that flowers raised in the green house is especially vulnerable to harsh conditions(固定搭配). 

Apparently(副词), this picture reveals to us(补语) that if we live in good environment, we might lose the courage to overcome difficulties. 

To sum up, due to the above mentioned reasons, we may safely arrive at the conclusion that difficulties are essential for our spirit.

2004
Sample
A boy, sweaty and happy, is dashing to embrace his victory in a race. However, when he crosses the finishing line, he, to our surprise(固定搭配), doesn’t cease. In front of his feet, there are two huge words-starting point.

Apparently(副词), this drawing reveals to us: a finishing line doesn’t announce the end of a race, but marks the starting of another one. This is true also in one’s life. One track represents one stage in the life journey, and the finishing line merely the realization of a short-term goal. To be the constant winner, continuous progress is needed(被动). 

The viability of this philosophy can be easily proved. Madame Curie, a world-renowned female scientist, was awarded twice the Nobel Prize for Physics and Chemistry, two totally different research fields. If she had stopped conducting further research, she would not have achieved unequalled success in the history of science. The spirit of keeping on striving is also shared by Chinese scientists. After short victory celebration of Shenzhou V spaceship’s launch, they continued developing Shenzhou VI and VII spaceships. There is also an example relevant to us. Upon graduation, we stand at a new starting point, either pursuing further study or starting a career. Only when(倒装) we are well-prepared for a new start, can we be the winner at the next stage.

2005
Part A
Sample
    I appreciate the opportunity of working here during these two months as an editor for the magazine DF, and particularly your constant assistance. However(连词), I’m writing to formally give notice of my resignation from my post due to personal reasons.
As a young man whose primary interest is in computer programming rather than fashion designing, I find my present job does not accord closely with my previous training and strength. Therefore(连词), I decide to vacate this job and find another one that better matches my educational background.
Please accept my sincere apology for any inconvenience my leaving may cause. I will do my utmost to assist in the hand-over progress.

Mine
Dear Mr. Wang,
    I appreciate the opportunity(定语) of working as an editor for this splendid magazine Design & Fashions two months ago. However, I am sorry to inform you that I have to quit due to the reason that I no longer love this job. 
    I feel very happy to work with you for 2 months and do hope that we still keep in touch in the future.
Best regards,
Li Ming

Part B
Sample
    In the picture above, a special football game is going on where four siblings as gatekeepers are trying hard to keep the ball, their old father, out of their gate, which represent their own home. It discloses a strange social phenomenon in contemporary Chinese society that adult children shirk responsibility for their aging parents.
Such deeds reflect the decline of the traditional virtue of filial piety. Respecting the old and caring for the young has been among the most cherished virtues since ancient times in China. However, as the society has grown more money-oriented(形容词) and people more self-interested(形容词), this virtue gets eroded along with distortion of ethical morality. How could one callous to his parents expect to be well-treated by his children when he grows old? How could it be possible for one to help other old people if he even disregards his won aging parents? The society will be threatened to be unstable if families, the basic units of a society, are not harmonious. 
How to change this unhealthy social ethos? We should vigorously advocate the traditional virtues of honoring parents and respecting the old and caring for the young in society. We need the power of ethical persuasion to encourage the young to feed the material and spiritual needs of the elderly. Furthermore, those who fail to support their elderly parents should be punished by law as well as condemned by morality. These efforts are all prerequisites for building a harmonious society in which senior(形容词) citizens are looked after properly(副词).

Mine
In this vivid(形容词) picture, an old father is acting as a football, while his four siblings(名词), acting as goal keeper, are trying hard to keep the ball out of their gates. Apparently(副词), the picture reveals to us that some children refuse to take the responsibility of taking care of their aging(形容词) parent, who especially need assistance from them.

These behaviour ought to be blamed for the following reasons. First and foremost(固定搭配), everyone is brought up by their parents. Therefore, they have the duty to take care of their parents after grown up. Moreover, old man often has difficulty in leading a orderly life. If they are not sent to the hospital in time when nobody is around, they might died of heart attack or other diseases. 

However, it should be confessed that some grown-ups are busy with their own life and have little time to accompany with their parent. Therefore, help from the community and government is also essential.

To sum up, due to the above following reason, we may safely arrive the conclusion that only with(倒装) concerted effort from all sides, can senior citizens in our society be(被动) properly(副词) looked after.

2006
Part A
Sample
Dear Sir or Madam,
Moved by the noble cause of Hope Project and encourage by what has been achieved, I would like to do my bit for the project by seeking a child as the recipient of my financial assistance. I’m writing to inquire about the possibility of finding a candidate.
I would be deeply grateful if you could help me seek out a girl who has just started schooling in a remote and poor area. My plan is to pay for her tuition fee on a yearly basis till she finishes her secondary education and additionally provide her with 200 yuan living expenses each month. I would like to regularly remit my donation directly to a bank account opened by her family.
Please contact me when you find a proper candidate. I really appreciate attention and hope to hear from you soon.

Mine
To whom it may concern,
Moved by(伴随状语) the noble cause of Hope Project and encouraged by what has been achieved, I would like to offer financial aid to a child in remote area.I’m writing to inquire about(动词) the possibility of find a candidate. 
I would be deeply grateful if(条件状语) you could help me seek out(寻找) a girl whose parents have passed away due to the following reasons. First and foremost, child whose parents died do not have financial support to get goo education, which will lead to poverty in the future. In addition, he or she might begin to love the society if receive help from strangers.
I really appreciate your attention(套话) and look forward to your reply at your earliest convenience.
Best regards,
Li Ming

Part B
Sample
As shown in the photos, one man has written “Beckham” on his face and the other spends 300 yuan imitating Beckham chic hairstyle. The photos reveal how fanatically devoted people are to their idols.
In this pluralistic and liberal era, idol-worshiping is prevalent among young generation. This obsession with celebrities, whether they are in the field of sports, entertainment or tech is harmless min itself, yet more often than not start-struck youngsters emphasize superficial stuff over substantial qualities, just like the two men in the pictures. They blindly copy Beckham’s looks without realizing that his achievements and international influence are the results of his hard work. Since he was a little boy, Beckham hung the tire on the woodwork to practice free kick over and over again. His success is attributed to his diligence and perseverance.
Handled properly, idol worship can bring the best out of us, serving as beacons to guide us towards our ideals. We should learn from luminaries’ combatant spirit, their firmness of will and unrelenting efforts and constantly improve ourselves by emulating those talented.

Mine
    There are two pictures above. The first picture depicts a man who paints the name of Beckham on his face, while in the second picture a male spends 300 yuan to imitate Beckham’s hairstyle. Apparently(副词), what this picture highlights(主语从句) is that young people nowadays show their love for sport stars in fanatic ways.  
    Reasons why this kind of phenomenon occurs from time to time are various(套话). One of possible causes is that(表语从句) sport stars always appear in the headlines of newspaper and website. It is also owing to the fact that(同位语) traditional virtues is longer attractive to them. 
    It’s true that reverse the situation is not an easy task, but a laissez-faire(形容词) attitude will invite even bigger trouble. First and foremost(连词), youngster must be taught what is right and what is wrong in school. In addition, website and newspapers ought to report more about scientists and teachers instead of sports celebrities. To sum up, only with concerted efforts of all individuals, can young generation be properly educated. 

2007
Part A
Sample
Dear sir or madam,
    I am a student in this university who frequently comes to the library. Generally speaking, the services you offer here are quite good. However, I have some suggestions for you to make the library more user-friendly.
    To begin with, will you(疑问) please consider installing self-checkout machines, which would render readers and librarians more convenience? Moreover, it is expected to(被动) adopt the online reading room reservation, which can reduce readers’ waiting line. In addition, we will be very pleased if you can provide opportunities for readers to express what we we want. The traditional suggestion box and its modern incarnation--an online suggestion form on the library’s homepage are terrific(形容词) ways to solicit this feedback.
I would like to extend my greatest appreciation if you are so kind as to take my suggestions into consideration.
Yours faithfully,
Li Ming

Mine
To whom it may concern,
As a student who frequently study in the library, I would like to give some suggestion to make the library more user-friendly.
First and foremost, would you please install air-conditioner, which would make library cool in the summer? In addition, it is expected(被动) to extend the open hours when mid-term and final examination is near. What is more, people who are not student of our school should not be allowed to enter since many students’ laptops are stolen in the library.
I extend my greatest appreciation for your attention and look forward to your reply at your earliest convenience(套话).
Best regards,
Li Ming.

Part B
Sample
The picture depicts a breathtaking moment when the player is about to shoot. At this time the goalkeeper in the player’s mind becomes a giant, who makes it incredibly difficult to score, while the goalkeeper imagines that he is a dwarf who can barely prevent the player from scoring a goal.
Both of them underestimate their abilities and overestimate their enemy’s. This kind of insecurity may lead to under-performance. The purpose of the drawing is to show us(宾语从句) that in the face of challenges, to be undaunted and self-assured is of great importance. There are tons of people who exude confidence all the time and have achieved a great deal, one of whom is Deng Yaping, the world famous table tennis player. Her short figure doesn’t stop her from becoming a legend in the table tennis field. Instead, her self-confidence as well as perseverance had enabled her to win not only 18 major world champions but also respect from all over the world.
Under the shadow of self-doubt, we may never fully release our potential or achieve success. Once we believe in ourselves, sparks of possibility may turn into flames of achievement. Just as Mark Twain said, all you need in this life is ignorance and confidence, and then success if sure.

Mine
In this vivid picture, a football player is standing in front of the goalkeeper, ready to shoot. In his mind, the goal keeper becomes a giant that cover the whole gate, while the goalkeeper treat himself as a dwarf even smaller than the ball. Apparently(副词), what this picture highlights(主语从句) is that some people tend to underestimate their abilities and overestimate their enemy’s. However, in the face of challenges, to be undaunted and self-assured is of great importance.
First and foremost, the course of history has informed us vividly that it is confidence that(强调句) give us courage to overcome difficulties, which is especially true in the realm of sports. A case in point(固定搭配) is the celebrated player Deng Yaping. Her short figure doesn’t stop her from becoming a length in the table tennis player. Instead, her self-confidence as well as perseverance had enabled her to win not only 18 major champions but also respect from all over the world.
On the other hand, a recent authoritative survey has unveiled that(宾从) we may never fully release our potential under the shadow of self-doubt. In the light of 20-year-study of 3000 candidate conducted by the Chinese Academy of Social Science, children who tended to underestimate their ability lived in poverty after grown up. They refused to try something new for fear that they would fail as they always did in childhood, despite the fact that(转折) some of them had high IQ and were well-educated.
    To sum up, due to the above mentioned reasoned, we may safely arrive to the conclusion that it is exceeding desirable, valuable and indispensable to have self-confidence in our way of life.

2008
Part A
Sample
Dear Bob,
I am writing to make an apology to you, for I forget to return the music CD which I borrowed from you last week. It is not until yesterday when I came back to Beijing that I found it in my luggage. I was busy preparing for my leave and it must have slipped in my mind.
To make up for my carelessness, I have mailed the CD back through EMS, along with a CD by Bob Dylan as a token of my apology. They will reach you in about one week.
Again, I would like to apologize for any inconvenience it may have caused to you. Looking forward to your reply.
                                                                    Yours,
                                                                    Li Ming
Mine
Dear Bob,
    I am your friend Li Ming who left Canada two weeks ago. I am sorry to inform you that I found your music CD in my luggage that I forgot to return to you.
    Can I send this CD back to your home through the UPS international mail system? 
    I give my sincere apology and look forward to your reply at your earliest convenience.
Best regards,
Li Ming

Part B
Sample
Walking stick thrown away and stumps bound together, two disable men, each of whom has only one leg, become a powerful union and are able to run forward as any able-bodied individual. Yes, what the picture reveals(主语从句) is the power of teamwork. It is no exaggeration to say, united we stand, divided we stagnate.
Teamwork is the spiritual power of working together toward a common vision and directing individual accomplishments toward organization objectives. A case in point is the Spurs, an NBA team. We’ve seen teams with expensive star players out-performed by this legendary team with players who may be individually less talented. The primary reason for its success is that they take teamwork, team chemisty, in the coach’s words, to an unparalleled level. Aiming at winning, Spurs players keep passing the ball and always give chance to those on the most favorable positions to shoot, thus creating its splendid records and the Spurs Dynasty in NBA history.
Collaboration can release and harness only your skills but also those of everyone on the team, thereby(连词) producing multiplying effect. So when adversity strikes, we’d better change our ways of thinking and regard cooperation as the fuel of brilliance. Active cooperation can change unfavorable conditions into favorable ones, turning the situation around and eventually working wonders(取得成功). This is the magic of cooperation!
Mine
    In this vivid picture, two disabled men, each of whom has only one leg, put their hand on the other’s shoulder and walk together after throwing away(搭配) their walking sticks, which are no longer needed since they can walk as healthy individuals(替代) do. Apparently(副词), what the picture highlights(主语从句) is the tremendous value cooperation, through which we can overcome obstacles in our life despite the fact that none of us is perfect. 
    The course of history has informed us vividly(宾从) that it is through cooperation that(强调句) we can learn from each other and go further in the way to success, which is especially true in the realm of sports. A case in point(固定搭配) is the celebrated(形容词) basketball player LeBron James. Although he had already gained the award of most valuable player long time ago, he did not get final champion until he joined Miami Heat and learned from his best friend, Dwyane Wade, who taught him how to pick and roll in a more efficient way. Armed with the new skills learned from Wade, LeBron James eventually(副词) beat Thunders in the NBA Final and got the award of FMVP.
To sum up, due to the above mentioned reasons, we may safely arrive at the conclusion that it is exceedingly desirable, valuable and indispensable for us to cherish the virtue of cooperation, which can deepen our friendship and ensure great achievements in the future.
I hope you will find the above proposals useful and call on the public to be more involved in environmental protection.
                                                            Yours sincerely,
                                                            Li Ming
2009
Part A
Sample
Dear Editors,
As a faithful reader of your newspaper, I always appreciate your insightful report of social issues. I am writing to express my concern over the abuse of plastic bags.
In some regions, plastic bags are still widely used despite restrictions by the government. To solve the problem, I believe it is imperative for the government to further control production and usage of plastic bags. In addition, I’d like to suggest that plastic bags be offered at a much higher price and greener alternatives be further promoted. Furthermore, newspapers should play a major role in advocating the importance of the restrictions and the concept of green consumption.

Mine
To whom it may concern,
As an important cause for economic losses and health risks, white pollution becomes a hot issue in our city. In the light of(替换) survey conducted by China Daily, seven out of ten citizens are bothered with plastic bags flying outside their apartments. I would like to offer some suggestions to prevent such undesirable(形容词) situation from going on.
First and foremost, the population of our city ought to be controlled within a reasonable range. In addition, laws and regulations should be adopted and enforced to restrict the use of(搭配) plastic bags. In summary, only with concerted efforts from all individuals, can the while pollution problem be properly solved.
Best regards,
Li Ming

Part B
Sample
As illustrated in the cartoon, an enormous spider web, which symbolizes the Internet, confines people to a tiny space with its virtual walls. Everyone is well within reach, but seems quite far apart. This is the very implication of nearness and farness of the Internet, which vividly reveals people’s life in today’s society.
For the past few years, instant messaging tools like QQ and WeChat have sprung up and practically been the most popular medium for interpersonal communication. Comparing with traditional letters in written....

Mine
In this cartoon(替换), individuals keep staying in their own room, chatting and interacting happily with their on-line friends. On the other hand, their neighbors are neglected despite the fact that they are only several meters away. Apparently(副词), what this picture highlights(主语从句) is that web enables us get in touch with friends far away, but prevents us from talking with our neighbors.
Reasons why this kind of phenomenon occurs are various(套话). One of possible causes is that compared with our neighbors, we can meet all of kinds of people on-line, which means we can select people who best match our characters to be our friends. It is also owing to the fact that(同位语) the internet can satisfy all our needs nowadays from ordering pizza to playing video games. As a result, the connection with neighbor is not as important as several decades ago.
    To prevent such undesirable situation from going on(套话), some actions need to be taken. First and foremost, take a break from screens and go out to take part in more social activities. Moreover, laws and regulations ought to be adopted and enforced to limit the playing time of on-line games. Only in this way can we maximize the advantage of the Internet and avoid its drawbacks(套话). 

2010
Part A
Sample
Mine

Part B
Sample
The pictures represents a steaming cultural hotpot with all kinds of foodstuffs in it, such as Buddhism, postmodernism, the Butterfly Lovers and Einstein. Mixing together, these cultural ingredients look tremendously delicious and nutritious. What lies behind the picture is clear: Multicultural integration is a noteworthy feature of the time and is of great benefit to(搭配) the inheritance and development of world culture.
We live in an unprecedented age of globalization....
Mine
In this picture, a steaming cultural hotpot is setting on fire and greeting people with pleasant smell. In the hotpot there are many kinds of foodstuffs(名词), which represents kongfu, Shakespear, Swan Lake and so on. Apparently(副词), what the cartoon highlights is that(主语从句) we need all kinds of “nutrition”, including foreign novels, modern technology and ancient religions, to make our culture attractive and fascinating to everybody.
The course of history has informed us vividly that(宾语从句) culture exchange is of great benefit to one nation. A case in point is the import of western culture during the great process of reform and opening-up. With more and more cultural products come to China, lovers can watch Titanic in the cinema and students can discuss NBA and Premier League in the class. On the other hand, Chinese food is enjoyed by more and more Canadians, and Chinese Kong Fong has become a symbol in the Hollywood. 
It should be confessed that culture exchange gives rise to some problem. Some youngsters burn the mid-night oil to watch football games, while others lost interest in traditional Chinese culture such as Peking Opera. In this sense, only with concerted efforts of all people(倒装), can we maximize the advantage of culture blending and avoid its drawbacks.
